src/kube-runtime/src/plugins/common/post.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
license header?
did you change permission for this file?

Reply:
All permissions are changed in code